the-telephone-app-fe/src/App.js

```diff
@@ -21,14 +28,18 @@ function App() {
     <>
       <Routes>
         <Route path={"/lobby"} element={<JoinRoom />}></Route>
-        <Route path={"/:id_room?"} element={<StartGame />}></Route>
+        <Route path="/:id_room?" element={<StartGame />} />
         <Route path={"/start"} element={<WriteSentence />}></Route>
         <Route path={"/draw"} element={<Draw />}></Route>
         <Route path={"/write"} element={<DescribePicture />}></Route>
         <Route path={"/book"} element={<ShowResult />}></Route>
+        <Route path={"/error"} element={<ErrorPage />}></Route>
+        <Route path={"/full"} element={<FullRoom />}></Route>
+        <Route path={"/cancel"} element={<CancelRoom />}></Route>
+        <Route path={"/exit"} element={<ExitGame />}></Route>
```

Comment:
Add a route at the bottom to redirect to error page if input path is not exist.
`<Route path="*" element={ErrorPage} /> `(try if this can be applied to the current react router version)